tests/plugins/uploadwidget/manual/paste.md

```diff
@@ -0,0 +1,38 @@
+﻿@bender-tags: 4.10.2, bug, clipboard, widget, filetools, 1217
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, uploadwidget, image2, uploadimage, uploadfile
+@bender-include: _helpers/xhr.js
+
+## No selection
+
+1. Copy an image.
```

Comment:
It would be nice to add here some image for copying. Especially for second test with multiple images.

Reply:
Agree, I will add some images. Note that they may look ugly thus required base64 format.